cross-modal-search/README.md

```diff
@@ -1,38 +1,26 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**
-
-- [Build a CrossModal Search System to look for Images from Captions and viceversa](#build-a-crossmodal-search-system-to-look-for-images-from-captions-and-viceversa)
-  - [Prerequisites](#prerequisites)
-  - [Prepare the data](#prepare-the-data)
-  - [Build the docker images](#build-the-docker-images)
-  - [Run the Flows](#run-the-flows)
-  - [Results](#results)
-  - [Documentation](#documentation)
-  - [Community](#community)
-  - [License](#license)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
-# Build a CrossModal Search System to look for Images from Captions and viceversa
-<p align="center">
- 
-[![Jina](https://github.com/jina-ai/jina/blob/master/.github/badges/jina-badge.svg "We fully commit to open-source")](https://jina.ai)
+# Build a CrossModal Search System to look for Images from Captions
```

Comment:
Can you make titles Title Case please?

Reply:
7821babc1884fffcc2d8076878ade4c02b8f0bc7